core/templates/check-in.html

```diff
@@ -36,7 +36,7 @@
 
         <%if (item.media !== null) {%> <td title='<%= item.media_type %>' ><%= item.media %></td> <% } else { %> <td></td> <% } %>
         <td><%= item.call_number %></td>
-        <td><%= item.name %></td>
+        <td title='Location: <%= item.location %>'><%= item.name %></td>
```

Comment:
Maybe this would be better if it only displayed if the location were known so that it doesn't display "Location: undefined."